src/content/fr/2019/mobile-web.md

```diff
@@ -0,0 +1,291 @@
+
+<figure>
+  <table>
+    <tr>
+      <th>Type</th>
+      <th>Fréquence (pages)</th>
+    </tr>
+    <tr>
+      <td>phone</td>
+      <td class="numeric">1,917</td>
+    </tr>
+    <tr>
+      <td>name</td>
+      <td class="numeric">1,348</td>
+    </tr>
+    <tr>
+      <td>textbox</td>
+      <td class="numeric">833</td>
+    </tr>
+  </table>
+  <figcaption>Figure 7. Types de saisie invalides les plus couramment utilisés</figcaption>
+</figure>
+
+Assurez-vous de bien vous informer et de renseigner les autres sur la grande quantité de types de saisie disponibles et vérifiez que vous n’avez pas de fautes de frappe, à l’image de des plus courantes, reprises dans la figure 7 ci-dessus.
+
+### Activation de l’autocomplétion pour les saisies
+
+L’attribut [`autocomplete`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) de l’élément `<input>` permet aux gens de remplir les champs du formulaire en un seul clic. Ils remplissent des tonnes de formulaires, souvent avec exactement les mêmes informations à chaque fois. Conscients de ce fait, les navigateurs ont commencé à stocker ces informations de manière sécurisée afin de pouvoir les réutiliser. Tout ce que les équipes de développement doivent faire, c’est utiliser cet attribut `autocomplete` pour indiquer aux navigateurs quelle est l’information exacte à remplir, et le navigateur fait le reste.
+
+<figure>
+  <div class="big-number">29,62&nbsp;%</div>
+  <figcaption>Figure 8. Pourcentage des pages utilisant <code>autocomplete</code>.</figcaption>
+</figure>
+
+Actuellement, seules 29,62 % des pages comportant des champs de saisie utilisent cette fonction.
+
+### Saisie des champs de mot de passe
+
+Permettre aux utilisateurs de copier et de coller leurs mots de passe dans votre page leur permet d’utiliser un gestionnaire de mots de passe. Les gestionnaires de mots de passe aident les utilisateurs à générer (et à mémoriser) des mots de passe forts et à les remplir automatiquement sur les pages web. Seulement 0,02&nbsp;% des pages web testées désactivent cette fonctionnalité.
+
+<p class="note">Note&nbsp;: Bien que cela soit très encourageant, il s’agit peut-être d’une sous-estimation en raison de l’exigence de notre <a href="./methodology">méthodologie</a> de ne tester que les pages d’accueil. Les pages intérieures, comme les pages de connexion, ne sont pas testées.</p>
+
+## Conclusion
+
+Pendant plus de 13 ans, nous avons traité le web mobile de façon accessoire, comme une vulgaire alternative aux sites pour ordinateurs de bureau. Mais il est temps que cela change. Le web mobile est maintenant _*le*_ web, et les sites pour ordinateurs de bureau tombent en désuétude. Il y a maintenant 4 milliards de smartphones actifs dans le monde, couvrant 70&nbsp;% de tous les utilisateurs potentiels. Qu’en est-il des ordinateurs de bureau&nbsp;? Ils sont actuellement 1,6 milliard et représentent de moins en moins d’utilisateurs du web chaque mois.
```

Comment:
_*le*_ => Gras et italique ???

Reply:
That's the way it was written in English. I went with it but yes, it may be hard to read.

Reply:
It's not bold and italic as it's just one of each. So it's Italic and Italic! This is how it renders:

```
...The mobile web is now <em><em>the</em></em> web, and...
```

Let's fix to just `*le*` and remove the unnecessary `_` before and after.

Also you took away the italics from `mobile` which is in the English. Was that intentional?

```suggestion
Pendant plus de 13 ans, nous avons traité le web *mobile* de façon accessoire, comme une vulgaire alternative aux sites pour ordinateurs de bureau. Mais il est temps que cela change. Le web mobile est maintenant *le* web, et les sites pour ordinateurs de bureau tombent en désuétude. Il y a maintenant 4 milliards de smartphones actifs dans le monde, couvrant 70&nbsp;% de tous les utilisateurs potentiels. Qu’en est-il des ordinateurs de bureau&nbsp;? Ils sont actuellement 1,6 milliard et représentent de moins en moins d’utilisateurs du web chaque mois.
```